test/widgets/test_chord.py

```diff
@@ -25,3 +28,68 @@ def test_chord_widget(fake_bar):
     hook.fire("enter_chord", "testcolor")
     assert chord.background == RED
     assert chord.foreground == BLUE
+
+
+def test_chord_persistence(manager_nospawn):
+    def no_op():
+        pass
+
+    class ChordConf(libqtile.confreader.Config):
+        auto_fullscreen = False
+        keys = [
+            KeyChord([], "a", [
+                Key([], "b", lazy.function(no_op))
+                ],
+                mode="persistent_chord"
+            ),
+            KeyChord([], "z", [
+                Key([], "b", lazy.function(no_op)),
+                ],
+                name="temporary_name"
+            )
+        ]
+        mouse = []
+        groups = [libqtile.config.Group("a"), libqtile.config.Group("b")]
+        layouts = [libqtile.layout.stack.Stack(num_stacks=1)]
+        floating_layout = libqtile.resources.default_config.floating_layout
+        screens = [
+            libqtile.config.Screen(
+                top=libqtile.bar.Bar([Chord()], 10)
+            )
+        ]
+
+    manager_nospawn.start(ChordConf)
```

Comment:
Defining a test class and then immediately passing it to `manager_nospawn.start()` is equivalent to defining the class at the module level and using it to parameterise the `manager` fixture... right? Is there any reason why we would favour one over the other? (Or have I missed something?)

Reply:
You've not missed anything!